src/vars/images.yml

```diff
@@ -5,8 +5,8 @@ foreman_container_image: quay.io/foreman/foreman
 foreman_container_tag: "{{ container_tag_stream }}"
 postgresql_container_image: quay.io/sclorg/postgresql-13-c9s
 postgresql_container_tag: "latest"
-pulp_container_image: quay.io/foreman/pulp
-pulp_container_tag: "foreman-{{ container_tag_stream }}"
+pulp_container_image: quay.io/evgeni/pulp
+pulp_container_tag: "nightly"
```

Comment:
Can be dropped once https://github.com/theforeman/pulpcore-packaging/pull/2118 is merged and the containers are rebuilt with it.